pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -17899,7 +17899,6 @@ self: {
        description = "Shell script analysis tool";
        license = stdenv.lib.licenses.gpl3;
        hydraPlatforms = stdenv.lib.platforms.none;
```

Comment:
@jonringer 
I believe this line has to be removed as well, otherwise hydra won't build this derivation.

However, this should happen next time the haskell packages are updated.

Reply:
That's probably the proper thing to do to get haskellPackages.ShellCheck, however, I just need it so that `pkgs.shellcheck` ran successfully